src/Compiler/Facilities/DiagnosticsLogger.fs

```diff
@@ -133,6 +133,8 @@ exception DiagnosticWithSuggestions of number: int * message: string * range: ra
 /// A diagnostic that is raised when enabled manually, or by default with a language feature
 exception DiagnosticEnabledWithLanguageFeature of number: int * message: string * range: range * enabledByLangFeature: bool
 
+exception ObsoleteDiagnostic of isError: bool * diagnosticId: string * message: string * urlFormat: string * range: range
```

Comment:
Like for Experimental, are we representing absence of information via `""` because this is what the attribute does in C# or when extracted at runtime?

Reply:
So yeah at runtime the absence of information is represented as `null` to I think it make sense to make this values optional `None` aka null